src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Badge/Badge.ts

```diff
@@ -87,14 +91,51 @@ export var bindPath = (
     var getBadge = (badge) => {
         return {
             title: badge.data[SITitle.nick].title,
-            path: badge.path
+            path: badge.path,
+            groups: badge.data[SIBadge.nick].groups
         };
     };
 
+    var getGroup = (group) => {
+        return {
+            title: group.data[SITitle.nick].title,
+            path: group.path
+        };
+    };
```

Comment:
I would expect a function called `getSomething()` to do HTTP. I know this is consistent with `getBadge()` though.